locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocationConfig.java

```diff
@@ -293,6 +293,12 @@
     public static final ConfigKey<Map<String,Object>> TEMPLATE_OPTIONS = ConfigKeys.newConfigKey(
             new TypeToken<Map<String, Object>>() {}, "templateOptions", "Additional jclouds template options");
 
+    @Beta
+    public static final ConfigKey<Boolean> USE_PUBLIC_ENDPOINT_AS_PRIVATE_ENDPOINT = ConfigKeys.newBooleanConfigKey(
+            "usePublicEndpointAsPrivateEndpoint",
```

Comment:
@Graeme-Miller I agree with @neykov that we should have a comment here to explain why it exists, and an example of when one would use it.